aptos-move/vm-genesis/src/lib.rs

```diff
@@ -729,6 +754,40 @@ fn create_and_initialize_validators_with_commission(
     );
 }
 
+fn create_pbo_delegation_pools(
+    session: &mut SessionExt,
+    pbo_delegator_configuration: &[PboDelegatorConfiguration],
+) {
+    let pbo_config_bytes = bcs::to_bytes(pbo_delegator_configuration)
+        .expect("PboDelegatorConfiguration can be serialized");
+    let mut serialized_values = serialize_values(&vec![
+        MoveValue::U64(90), // Delegation % - hardcoded for prototyping
+    ]);
+    serialized_values.insert(0, pbo_config_bytes);
+    exec_function(
+        session,
+        GENESIS_MODULE_NAME,
+        "create_pbo_delegation_pools",
+        vec![],
+        serialized_values,
+    )
+}
+
+fn create_vesting_without_staking_pools(
+    session: &mut SessionExt,
+    vesting_pools_map: &[VestingPoolsMap],
+) {
+    let serialized_values =
+        vec![bcs::to_bytes(vesting_pools_map).expect("VestingPoolsMap can be serialized")];
```

Comment:
Did you mean `can't be serialized`? Since I believe you want to print this as an error message as an argument to `expect`.

Reply:
As a best practice in Rust, `expect` error messages are supposed to explain _why_ the outcome should work. So here, We _**expect**_  the `VestingPoolsMap` type to be successfully serializable _**ideally**_, but it is failing.

Eg: https://doc.rust-lang.org/std/option/enum.Option.html#recommended-message-style